tests/test_fhir.py

```diff
@@ -37,29 +37,23 @@ def test_codeable_concept_codings(self):
         cc = db.session.merge(cc)
 
         # now parse a fhir snippet containing first just a partial set
-        data = {"test_concept1":
-                {"coding": [{"system": initial_coding1.system,
-                             "code": initial_coding1.code,
-                             "display": initial_coding1.display}, ],
-                },
-               }
+        data = {"test_concept1": {"coding": [{
```

Comment:
This obfuscates the nesting. Can you put `coding` on its own line, and indent its contents beneath it?